tables/scripts/ptrepack.py

```diff
@@ -422,8 +422,15 @@ def main():
         )
 
     # Catch the files passed as the last arguments
-    src = args.src.split(':')
-    dst = args.dst.split(':')
+    if 'win' in sys.platform():
+        # the file separator for windows is deemed ','
+        src = args.src.split(',')
+        dst = args.dst.split(',')
+    else:
+        # the file separator for osx, linux for  is deemed ':'
+        src = args.src.split(':')
```

Comment:
If ptrepack arguments are sourcefile and destfile, yes I agree.  

If on Windows and ptrepack arguments are sourcegroup and destgroup each a sequence of absolute paths starting with a drive letter e.g. F:/path/to/file then it seems to me splitting on ":" won't work. But personally I always use ptrepack with sourcefile and destfile. Not sure it is a frequent case...

Anyway you're the maintainer and decide. So I've updated the PR as you suggested.   
I'll wait to see how you write the test to learn how for next time.  

Thx